src/session_pool/session.js

```diff
@@ -0,0 +1,171 @@
+import EventEmitter from 'events';
+
+import { cryptoRandomObjectId } from 'apify-shared/utilities';
+import moment from 'moment';
+
+import { checkParamOrThrow } from 'apify-client/build/utils';
+import EVENTS from './events';
+
+
+/**
+ *  Class aggregating data for `Session`.
+ *  Session internal state can be enriched with custom user data for example some authorization tokens.
+ */
+export default class Session {
+    constructor(options = {}) {
+        const {
+            id = cryptoRandomObjectId(),
```

Comment:
Let's prefix it with `"session-"`

Reply:
And make the ID shorter than 17 chars, that's not necessary for ID. 10 chars are enough